src/transformTs.js

```diff
@@ -135,6 +135,41 @@ function checkNestedObjectByKeyValue(obj, objKey, objValue) {
     return false
 }
 
+function convertTimeUnixNano(metric){
```

Comment:
Why do we need to add this logic when you can just set `time.now()` for every sample? https://github.com/logzio/js-metrics/blob/main/src/transformTs.js#L4